update.php

```diff
@@ -150,4 +150,4 @@
       <td><input type="submit" name="update" Value="Update Doggo"></td>
     </tr>
   </table>
-</form>
\ No newline at end of file
+</form>
```

Comment:
Řádek 19 - u této appky by už se hodilo vytvořit soubor např. routes.php a tam ukládat všechny routy aplikace, abys pořád nemusela v kódu měnit routy v případě potřeby (momentálně je kód složitěji upravitelný).

Je možné, že se nějaký kód opakuje.. např. toto:

```
while ($row = $result->fetch_assoc()) {
      if ($row['dog_id'] == $dog_id) {
        $dogFound = true;
        break;
      }
    }
```
V takovém případě je super vytvářet pomocné funkce :+1: 

Opět složité IFy, funkce by pomohly.

Tato funkce je pěkná a už jsem jí jednou viděl. Všiml jsem si, že nesedí možnosti pro věk psa.. opět by bylo fajn funkci někde vytáhnout a mít jedno místo, kde tuto funkcionalitu můžeš upravit.